frontend/src/pages/BYONImages/BYONImagesTable.tsx

```diff
@@ -6,8 +6,8 @@ import DashboardSearchField, { SearchType } from '~/concepts/dashboard/Dashboard
 import DashboardEmptyTableView from '~/concepts/dashboard/DashboardEmptyTableView';
 import { useDashboardNamespace } from '~/redux/selectors';
 import useAcceleratorProfiles from '~/pages/notebookController/screens/server/useAcceleratorProfiles';
-import ManageBYONImageModal from './BYONImageModal/ManageBYONImageModal';
-import DeleteBYONImageModal from './BYONImageModal/DeleteBYONImageModal';
+import { ManageBYONImageModal } from './BYONImageModal/ManageBYONImageModal';
+import { DeleteBYONImageModal } from './BYONImageModal/DeleteBYONImageModal';
```

Comment:
We want to import components as default imports and not named. We should remove the named export in this case and all other cases found in this PR.